charts/openfaas-cloud/values.yaml

```diff
@@ -0,0 +1,73 @@
+# Default values for openfaas-cloud.
+# This is a YAML-formatted file.
+
+ofBuilder:
+  image: openfaas/of-builder:0.7.2
+  replicaCount: 1
+
+buildKit:
+  image: moby/buildkit:v0.6.2
+  privileged: true
+
+
+edgeAuth:
+  image: openfaas/edge-auth:0.6.3
+  replicaCount: 1
+  oauthProvider: github
+  ## OAuthProviderBaseURL can be left blank for github, Gitlab users need to provide the url for their gitlab instance
+  oauthProviderBaseURL:
+  ## clientId is generated by your Oauth App, and needs to be set to connect correctly
+  clientId:
+  writeDebug: false
+  ## clientSecret is the value from your OAuth application
+  clientSecret:
+
+edgeRouter:
+  image: openfaas/edge-router:0.7.4
+
+tls:
+  enabled: false
+  ## email is used to send cert-renewal emails, it is required by LetsEncrypt
+  email:
+
+  ## Default to using the staging issuer, there's a limit to 5 certs per domain on the prod issuer, so only when you are
+  ##  happy should you swap this to prod
+  issuerType: staging
+
+  ## dnsService, one of "route53", "digitalocean" , "cloudflare" or "clouddns"
+  dnsService:
```

Comment:
How do you "select" a provider?

Reply:
--set dnsService=route53

We could make an assumption, but that assumption assumes they have created the correct secrets ?